modules/shapefile/test/data/README.md

```diff
@@ -0,0 +1,3 @@
+# Data Attribution
+
+- Files in the `bostock/` folder are from `mbostock/shapefile`
```

Comment:
Specify what license (add it to the folder). Also his license forbids the use of his name, maybe pick another directory name :)